source/administration/devops-command-center.rst

```diff
@@ -62,50 +62,124 @@ The set of steps contained within a playbook is called a checklist.
+3. Enter a name for the playbook.
+4. Move the toggle to the right to **Create Public Incident**.
+5. Add a few checklist items.
+6. Select **Save**.
+7. Switch to the main channel view.
+8. Create an incident by selecting the playbook you just created.
 
-You can add steps and edit checklist items during an active incident. You can also edit checklist items in playbooks that are not being used in active incidents.
+**Creating a private playbook and incident**
 
-To edit a checklist, choose **Edit**.
+1. Navigate to **Main Menu > Incidents & Playbooks > Playbooks**.
+2. Select **+ New Playbook**.
+3. Enter a name for the playbook.
+4. Keep the **Create Public Incident** option disabled.
+5. Add a few checklist items.
+6. Select **Save**.
+7. Switch to the main channel view.
+8. Create an incident by selecting the playbook you just created.
 
-- Select the text field to edit the item details.
-- Hover over an input box and select the **X** to remove the checklist item.
-- Drag the hamburger menu to rearrange the checklist items.
+Editing checklists
+~~~~~~~~~~~~~~~~~~~
+
+You can edit checklist items and their position in the checklist when they are not part of an active incident.
 
-Choose **Done** to save the changes.
+1. Select **!** in the channel header of an active incident.
+2. Choose **Edit**.
+  - To edit: Select the text field to edit details.
+  - To delete: Hover over an input box and select the **X** to remove the checklist item.
+  - To move: Drag the hamburger menu to rearrange the checklist items.
+3. Choose **Done** to save the changes.
 
 Ending incidents
 ~~~~~~~~~~~~~~~~
 
 There are two ways to end an incident:
 
-- Using slash command */incident end* from within the incident channel.
+- Using the slash command */incident end* from within the incident channel.
 - Using the **End Incident** button in the RHS panel while in the incident channel.
 
-The incident will become inactive, be removed from the list of active incidents, and the associated channel will be archived.
+The incident will become inactive, is removed from the list of active incidents, and the associated channel is archived.
 
 Administrator's Guide
 ^^^^^^^^^^^^^^^^^^^^^^
 
+Incidents and playbooks are associated with teams in Mattermost. Participants can be added to an incident by other incident members inviting them to the incident's channel. Incidents are created based on playbooks, and are public or private depending on the playbook's settings. 
+
 Permissions
 ~~~~~~~~~~~~~~~~~~~~~
 
-Incidents and playbooks are associated with teams in Mattermost. Participants can be added by inviting them to the incident's channel.
+System Admins and Team Admins can edit the visibility of, and access to, playbooks and incidents so that:
+
+- Users who are removed from an incident channel by a System Admin can no longer access the incident's playbook.
+- Team members who aren't System or Team Admins can invite other team members to manage private playbooks.
```

Comment:
Team Admins currently have no special privileges within the incident response domain model, so let's remove all references to them for now.